src/app/pages/MarginTradePage/components/OpenPositionsTable/ProfitContainer.tsx

```diff
@@ -134,13 +134,13 @@ export const ProfitContainer: React.FC<ProfitContainerProps> = ({
               <div className="tw-mt-1 tw-pl-3">
                 <AssetValue
                   asset={loanToken}
-                  value={toWei(exitAmountLoan.value.withdrawAmount)}
+                  value={exitAmountLoan.value.withdrawAmount}
                 />
               </div>
               <div className="tw-pl-3">
                 <AssetValue
                   asset={collateralToken}
-                  value={toWei(exitAmountCollateral.value.withdrawAmount)}
+                  value={exitAmountCollateral.value.withdrawAmount}
```

Comment:
I would adjust it to `maxDecimals={8} mode={AssetValueMode.auto}`, otherwise I see zero amounts in some of my positions (and I verified that the amounts are non-zero).